WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCouponViewModel.swift

```diff
@@ -241,6 +241,7 @@ final class AddEditCouponViewModel: ObservableObject {
             case .success(_):
                 ServiceLocator.analytics.track(.couponUpdateSuccess)
                 self.onCompletion(result)
+                onUpdateFinished()
```

Comment:
I added this closure parameter here to allow us to dismiss the `EditAddCoupon` view after the update ends, since we removed the sheet observing the `showingEditCoupon` property state. WDYT?

Reply:
This makes sense too! 👍